fr/index.html

```diff
@@ -0,0 +1,391 @@
+          </p>
+        </section>
+
+        <section class="myth">
+          <h2 class="myth__title">L'accessibilité peut être obtenue en ajoutant uniquement des attributs ARIA</h2>
+
+          <p>
+            Bien sûr, <a href="https://www.w3.org/WAI/standards-guidelines/aria/" target="_blank">ARIA</a> constitue une
+            part importante de l'accessibilité, mais il ne faut jamais oublier la règle d'or d’ARIA:
+            <a href="https://w3c.github.io/using-aria/#rule1" target="_blank"
+              >N'utilisez pas ARIA que si vous en avez vraiment besoin</a
+            >. De nombreux attributs ARIA peuvent ne pas être nécessaires si
+            <a
+              href="https://developer.mozilla.org/fr/docs/Learn/Accessibility/HTML#une_bonne_s%C3%A9mantique"
+              target="_blank"
+              >des balises HTML sémantiques</a
+            >
+            sont utilisées à la place.
+          </p>
+        </section>
+
+        <section class="myth">
+          <h2 class="myth__title">Nous n'avons pas besoin d'être conformes</h2>
+
+          <p>
+            Peut-être que vous en avez besoin. Il existe un certain nombre de lois et de règlements qui peuvent exiger
+            qu'un site Web respecte les principes d'accessibilité. Cela vaut assurément la peine de vérifier.
+          </p>
+
+          <a href="https://www.w3.org/WAI/policies/" target="_blank"
+            >Plus de détails <span class="visually-hidden"></span
+          ></a>
+        </section>
+
+        <section class="myth">
+          <h2 class="myth__title">Seuls des experts en accessibilité peuvent corriger des problèmes d'accessibilité</h2>
+
+          <p>
+            Heureusement, non. Toute personne ayant quelques connaissances en matière d'accessibilité peut mettre en
+            œuvre et tester l'accessibilité des sites web. En fait, non seulement c'est facile à apprendre, mais c'est
+            parfois un processus assez amusant. Il n'est pas nécessaire d'avoir une expérience spécifique pour effectuer
+            <a href="https://www.w3.org/WAI/test-evaluate/preliminary/" target="_blank">les vérifications de base</a>.
+          </p>
+        </section>
+
+        <section class="myth">
+          <h2 class="myth__title">
+            L'accessibilité, c’est comme une fonctionnalité - une fois que c’est fait, c’est fait
+          </h2>
+
+          <p>
+            Non. L'entretien quotidien, les nouveaux collègues qui n'ont pas les compétences requises en matière
+            d'accessibilité, les nouvelles exigences, tout cela peut compromettre l'accessibilité. L'accessibilité est
+            un élément incontournable qui doit être pris en compte à chaque étape du développement.
+          </p>
+
+          <a href="https://ethanmarcotte.com/wrote/accessibility-is-not-a-feature/" target="_blank"
+            >Plus de détails <span class="visually-hidden"></span
```

Comment:
@MarionCouesnon could you please add a myth name to `visually-hidden`.

Reply:
Done with 77c2a05d2459067877887858494997c996167cdd